components/delivery-city.jsx

```diff
@@ -22,8 +22,14 @@ export function DeliveryCity ({
 		CAN: 'e.g. Montreal',
 		ARE: 'e.g. Dubai'
 	};
+	const FTShippingZone = identifyFTShippingZone(country);
 
-	const inputLabel = country === 'ARE'? 'Emirate or City':'City';
+	let inputLabel = 'City';
+	if(FTShippingZone === 'ARE'){
+		inputLabel = 'Emirate or City';
+	} else if (FTShippingZone === 'APAC'){
+		inputLabel ='City/Prefecture';
+	}
```

Comment:
Nothing wrong with your version (don't feel like you have to change this), just an alternative, I think closer to what is just above with the placeholder mapping, and easier to extend:

```suggestion
	const labelMapping = {
		ARE: 'Emirate or City',
		APAC: 'City/Prefecture',
		DEFAULT: 'City',
	};
	const inputLabel = labelMapping[FTShippingZone] ? labelMapping[FTShippingZone] :  labelMapping['DEFAULT]';
```

Reply:
Cunning! 🌟